qtensor/contraction_backends/torch.py

```diff
@@ -64,7 +68,7 @@ def get_sliced_buckets(self, buckets, data_dict, slice_dict):
                 transpose_order = np.argsort(list(map(int, tensor.indices)))
                 data = data_dict[tensor.data_key]
                 if not isinstance(data, torch.Tensor):
-                    data = torch.from_numpy(data)
+                    data = torch.from_numpy(data).to(cuda)
```

Comment:
Can you add an option in the Torch backend to specify where to put the tensors? We might need both CPU and GPU torch, so user has to be able to specify which.
Also in the function `get_backend(str)` we should have `torch`, `torch_cpu` and `torch_gpu`, with `torch` being cpu by default.